ocfweb/stats/semester_job.py

```diff
@@ -0,0 +1,109 @@
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+
+import ocflib.printing.quota as quota
+from django.http import HttpResponse
+from matplotlib.figure import Figure
+from matplotlib.ticker import MaxNLocator
+from ocflib.printing.quota import WEEKDAY_QUOTA
+from ocflib.printing.quota import WEEKEND_QUOTA
+
+from ocfweb.component.graph import canonical_graph
+from ocfweb.component.graph import plot_to_image_bytes
+
+ONE_DAY = timedelta(days=1)
+MONTH_INDEX = 1
+SEMESTER_CUTOFF_MONTH = 8
```

Comment:
we have some logic for this elsewhere in ocflib, let's try not to duplicate how we determine what the current semester is even more than we already do